### DISCUSSION TITLE
求助：打游戏协议求推荐

### DISCUSSION THREAD

**Discussion by lhbdhr** [XTLS/Xray-core#4173] [Q&A] [answered]
最近迷上了PUBG，
方案一：使用 VLESS+TCP+VISION，感觉TLS对延迟影响很大啊，
方案二：还尝试了VLESS+REALITY+GRPC，不是多路复用吗？我理解中应该会减少握手次数吧？但是延迟几乎和 第一种没区别，
方案三：直接用 socks5明牌，v2rayn TUN模式，仅打开PUBG，其他软件都关掉。 竟然会被断流。客户端直接 curl --socks ip.gs 本来是能返回ip的，一打开PUBG，直接断流几分钟，每次都是。
方案四：ss？ 没敢试，不是明文过墙，谁知道你是不是在打游戏啊，感觉会被秒封端口。

我搜issue的时候，好像R佬说过会开发一个专门给游戏用的协议啊，敲碗。。。

**Comment by netfishclub** [comment]
ss 还行吧，~~打游戏人家一般不管~~，其实还是建议用专门的加速器，服务有保障。我socks5 都没设密码（~~软件不支持有密码的socks5~~），直接拿来跑挖矿 xmrig ，也没出现断流和被封的情况。

**Reply by netfishclub** [reply]
如果你买服务器只是为了打游戏，那其实没必要的，国内加速器速度和价格都还可以，
如果你又要科学上网，又要加速游戏，选择KCP协议降低延迟,如果打游戏可以尝试该协议（来源于网络，没试过）

**Reply by lhbdhr** [reply, marked as answer]
> 如果你买服务器只是为了打游戏，那其实没必要的，国内加速器速度和价格都还可以， 如果你又要科学上网，又要加速游戏，选择KCP协议降低延迟,如果打游戏可以尝试该协议（来源于网络，没试过）

感谢，
我后来尝试了 socks5，可能之前黑名单软件没关干净，这次没断流，结果发现和 tls+vision 延迟差不多。
我最后发现，是我的节点和游戏服务器之间的延迟太高。我到节点40ms，节点到游戏服务器40+ms，属于线路问题.

**Comment by ilafz** [comment]
用Netch grpc协议延迟很低

**Comment by doughnutscranberry** [comment]
2025了 现在打游戏用什么协议，要延迟低和稳定，就是普通线路。不期待能降低延迟 但是也别开个协议就增加50ms延迟
